### ISSUE TITLE
[JOSS Review] Enhance csv file instructions

### ISSUE THREAD

**Issue by danasolav** [SwathiSheshadri/pose3d#7]
The instructions should clearly explain how to create the csv file in case the 2D points are not generated using DLC. In my opinion, asking the user to use a demo file as a reference and learn from it is insufficient.

**Comment by SwathiSheshadri** [comment]
Thank you for pointing this out! The readme file is now up-to-date with details on how to prepare 2D tracked data for pose3d when using tracking software other than DLC. Specifically, we have updated the section 'Data requirements for pose3d' on the readme file to provide clear guidelines to users.

**Comment by danasolav** [comment]
Thank you for the prompt response. I was able to prepare the csv and run it, but encountered the following error:
Struct contents reference from a non-struct array object.
```
Error in load_preprocess_2d_data>load_otherdata (line 207)
    data2d = data2d.data;
```
The reason is that when my csv file is imported using importdata, it does not create a structure with a data field, it creates a matrix (possibly because my csv file does not contain any text or headers).

In addition, I have a minor comment about the way these guidelines appear in the readme file- it seems as if it is a part of point number 3, which is about calibration. I think the instructions for creating the csv file should be given in a new bullet point.

**Comment by SwathiSheshadri** [comment]
Thank you for your quick response and for giving me the details of the error. The latest release of the repository v2.0.0 includes a check to see if the variable is a structure and references the data field afterwards. I included this check in the code, since I anticipated that some users might have text fields in their .csv files and some others not. Could you please let me know if this error still exists when running the latest code available at the repository (released as version v2.0.0)?

**Comment by danasolav** [comment]
Thanks. I checked now and it works in the new version.
